<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcpClientFileSend</class>
 <widget class="QMainWindow" name="tcpClientFileSend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>桌面传输系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="gSource.qrc">
      <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>250</width>
      <height>250</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="clientStatusLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>325</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>客户端就绪</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="gSource.qrc">
      <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHost">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>271</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>startButton</zorder>
   <zorder>clientStatusLabel</zorder>
   <zorder>disconnectButton</zorder>
   <zorder>lineEditHost</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gSource.qrc"/>
 </resources>
 <connections/>
</ui>
